app/src/main/kotlin/com/twain/interprep/presentation/ui/components/InterviewCard.kt

```diff
@@ -0,0 +1,206 @@
+package com.twain.interprep.presentation.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.CardDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import com.twain.interprep.presentation.ui.theme.Shapes
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.ElevatedCard
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.twain.interprep.data.model.InterviewStatus
+import com.twain.interprep.presentation.ui.theme.BrightGreen
+import com.twain.interprep.presentation.ui.theme.DarkGray
+import com.twain.interprep.presentation.ui.theme.DateTimeGray
+import com.twain.interprep.presentation.ui.theme.Gray
+import com.twain.interprep.presentation.ui.theme.Green
+import com.twain.interprep.presentation.ui.theme.LightGray
+import com.twain.interprep.presentation.ui.theme.LightGreen
+import com.twain.interprep.presentation.ui.theme.LightPurple
+import com.twain.interprep.presentation.ui.theme.Primary30
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+val calendar: Calendar = Calendar.getInstance()
+val interview = Interview(
+    interviewId = 1,
+    date = calendar,
+    company = "Uber",
+    interviewType = "In-person",
+    role = "Software Engineer",
+    roundNum = 2,
+    jobPostLink = "https://www.linkedin.com/jobs/collections/recommended/?currentJobId=3512066424",
+    companyLink = "https://www.uber.com/ca/en/ride/",
+    interviewer = "John Smith",
+    interviewStatus = InterviewStatus.NEXTROUND
+)
+
```

Comment:
To focus on implementing composable, I created this instance of `Interview`

Reply:
Is it temporary and will be removed after the DB implementation will be done?

Reply:
Yes, I will remove it once the DB implementation is done.